tests/Microsoft.Graph.DotnetCore.Core.Test/Requests/Content/BatchRequestContentTests.cs

```diff
@@ -189,8 +194,10 @@ public async System.Threading.Tasks.Task BatchRequestContent_GetBatchRequestCont
         [Fact]
         public async System.Threading.Tasks.Task BatchRequestContent_GetBatchRequestContentFromStepAsyncDoesNotModifyDateTimes()
         {
+            // System.Text.Json is strict on json content by default. So make sure that there are no 
+            // trailing comma's and special characters
             var payloadString = "{\r\n" +
-                                "  \"subject\": \"Let's go for lunch\",\r\n" +
+                                "  \"subject\": \"Lets go for lunch\",\r\n" +
```

Comment:
What if someone was passing it in as "Let's".  Is there code now broken?

Reply:
No.The serializet will however encode the the `'` character. I only removed it to simplify the test.